### ISSUE TITLE
[Suggestion] Allow `chotbar` more than 9

### ISSUE THREAD

**Issue by barraIhsan** [Earthcomputer/clientcommands#248]
So, it's basically just allowed `chotbar` more than 9. That's mean that there is an infinite empty slot in the saved toolbar (scroll infinitely)? No
It **add empty slot** if needed. For example, when you type `chotbar save 20`, then, slot 10 to 20 unlocked (that's mean you can scroll until slot 20)

**Comment by xpple** [comment]
You essentially want a scrollable hotbar tab?

**Comment by barraIhsan** [comment]
No, I mean the usual tab. Because I never do modding the game. I taught that rendering so many slot is laggy. So it doesn't lag at all?

**Comment by xpple** [comment]
So you want to be able to have more than 9 hotbar slots?

**Comment by barraIhsan** [comment]
Yes
